### DISCUSSION TITLE
Github Dimmed theme changes

### DISCUSSION THREAD

**Discussion by madrussa** [community/community#51931] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Why has the text contrast changed so much on the dimmed theme? It doesn't need to be accessible with a high contrast if it's not the default. The searing white makes the "dimmed" aspect pointless. 

I am all for accessible and good contrast design but only where the user doesn't have an alternative. 

The change to the theme is disappointing and also makes the syntax highlighting less readable with bright green backgrounds and bright white text.

**Comment by dbeckwith** [comment]
I wonder if it's unintentional? The main text color on Dark Dimmed is now `#e6edf3`, and on Dark Default it's `#e4e8ec`. Dark Dimmed is brighter than Dark Default, which doesn't seem right.

**Reply by dbeckwith** [reply, marked as answer]
Seems to have been reverted; the Dimmed Dark color is back to `#adbac7` :tada:

**Reply by madrussa** [reply]
I can stop wearing sunglasses 😎
